db/03-employee-tables.sql

```diff
@@ -4,9 +4,9 @@
 USE MapBook;
 
 CREATE TABLE IF NOT EXISTS Course_Employee (
-    course_id VARCHAR(30) NOT NULL,
+    course_id INT NOT NULL,
```

Comment:
This should be in migration 04, but the table will get deleted there anyway so don't worry about it.
```suggestion
    course_id VARCHAR(30) NOT NULL,
```